corehq/apps/export/tests/test_export_data_schema.py

```diff
@@ -28,9 +28,11 @@ def test_basic_xform_parsing(self):
 
         group_schema = schema.group_schemas[0]
 
-        self.assertEqual(len(group_schema.items), 2)
-        self.assertEqual(group_schema.items[0].path, ['data', 'question1'])
-        self.assertEqual(group_schema.items[1].path, ['data', 'question2'])
+        self.assertEqual(len(group_schema.items), 2 + len(MAIN_TABLE_PROPERTIES))
```

Comment:
all of these + `len(MAIN_TABLE_PROPERTIES)` account for the additional props added to the schema for system properties